packages/ui/src/components/filters/CheckboxFilter.tsx

```diff
@@ -121,9 +118,13 @@ const CheckboxFilter: React.FC<TermFilterProps> = ({
                             type="text"
                         >
                             {isShowingMore
-                                ? get(dictionary, 'actions.less', 'less')
-                                : filteredFilters.length - 5 &&
-                                  `${filteredFilters.length - 5} ${get(dictionary, 'actions.more', 'more')}`}
+                                ? get(dictionary, 'actions.less', 'Less')
+                                : filteredFilters.length - 5 && (
+                                      <>
+                                          <>{`${filteredFilters.length - 5} `}</>
```

Comment:
what is the need for child `<>`?

Reply:
get fonction can return a React Element so I cant put it inside de `` so to display the return correctly from the get I need to put it inside his own "react element"